rdtools/test/analysis_chains_test.py

```diff
@@ -196,12 +223,21 @@ def test_filter_components_no_filters(sensor_parameters):
     assert rd_analysis.sensor_filter_components.empty
 
 
+def test_daily_filter_components_no_filters(sensor_parameters):
+    rd_analysis = TrendAnalysis(**sensor_parameters, power_dc_rated=1.0)
+    rd_analysis.filter_params = {}  # disable all index-based filters
+    rd_analysis.daily_filter_params = {}  # disable all daily filters
+    rd_analysis.sensor_analysis(analyses=['yoy_degradation'])
+    expected = pd.Series(True, index=rd_analysis.pv_energy.index)
+    pd.testing.assert_series_equal(rd_analysis.sensor_filter, expected)
+    assert rd_analysis.sensor_filter_components.empty
```

Comment:
I think these lines didn't get updated and are still checking the high frequency filter

Reply:
Updated to check aggregated filter in https://github.com/NREL/rdtools/pull/348/commits/bd6084faad5b67a8c0302f055450a4d47468dc77